http/src/request/guild/create_guild.rs

```diff
@@ -111,20 +213,49 @@ impl<'a> CreateGuild<'a> {
 
     /// Set the channels to create with the guild.
     ///
-    /// The maximum number of channels that can be provided is 500.
+    /// The maximum number of channels that can be provided is 500. A `Vec` of
```

Comment:
We can just say "list" here, it's in the signature. Specifying "\`Vec\`" just reads too programmatic